src/addresstype.h

```diff
@@ -23,6 +23,15 @@ class CNoDestination {
     friend bool operator<(const CNoDestination &a, const CNoDestination &b) { return a.script < b.script; }
 };
 
+struct PubKeyDestination
```

Comment:
in https://github.com/bitcoin/bitcoin/pull/28246/commits/34d7ab05b3f30e2827d9358a212f25865a71cce8:

Looks like this is causing the fuzzer to fail in `src/test/fuzz/util.cpp` because this new type is not handled.

Reply:
Should be fixed.